snyk/cli
# feat: Implemented new issue description [CFG-1574]

#### What does this PR do?

- Implements the new issue description design for the Snyk IaC test output.
- Applies the new issue description design in the test flow, wrapped in the `iacCliOutput` feature flag.

#### Where should the reviewer start?

- `src/lib/formatters/iac-output/v2/issues-list/index.ts`

#### How should this be manually tested?

- Execute the `snyk iac test` command without the `iacCliOutput` feature flag, and ensure the output is the same as the original.
- Execute the `snyk iac test` command with the `iacCliOutput` feature flag, ensure the issues list is displayed, and each issue is displayed with the expected values and [design] (https://www.figma.com/file/buNFZqr1Uhf4SaifHw4WXE/CLI-IaC-Output-Improvements?node-id=0%3A1).

#### Any background context you want to provide?

- As part of the MVP for the new CLI output, we’d like to replace the current details section offered per issue, with a new, more descriptive one. The new details added to this section will allow users to get additional information about these issues, as well as guidance on how to resolve them.

#### What are the relevant tickets?

- [CFG-1574](https://snyksec.atlassian.net/browse/CFG-1574)

#### Screenshots

- Before
    ![image](https://user-images.githubusercontent.com/46415136/163718228-d8ef33f8-8f62-43d8-8ad1-638785de8771.png)


- After
    ![image](https://user-images.githubusercontent.com/46415136/163718468-cabef078-d241-4ba6-9c2f-ed1c7d237e9e.png)
